### ISSUE TITLE
Chocolatey Provider should return only one entry to Oneget's Install-Package

### ISSUE THREAD

**Issue by JohnLudlow** [chocolatey-community/chocolatey-oneget#6]
I tried to install VirtualBox using the Windows 10 PackageManagement Install-Package cmdlet, but got this error:

``` powershell
PS C:\WINDOWS\system32> install-package virtualbox
WARNING: 'virtualbox' matched package 'virtualbox/5.0.14.105127' from provider: 'Chocolatey', source 'chocolatey'
WARNING: 'virtualbox' matched package 'VirtualBox.ExtensionPack/5.0.14.105127' from provider: 'Chocolatey', source
'chocolatey'
install-package : Unable to install, multiple packages matched 'virtualbox'. Please specify an exact -Name and
-RequiredVersion.
At line:1 char:1
+ install-package virtualbox
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (Microsoft.Power....InstallPackage:InstallPackage) [Install-Package], E
   xception
    + FullyQualifiedErrorId : DisambiguateForInstall,Microsoft.PowerShell.PackageManagement.Cmdlets.InstallPackage


PS C:\WINDOWS\system32>
```

Clearly, I want to install VirtualBox. I got around this by using `Find-Package` and piping into `Install-Package`, but this shouldn't really be necessary. I noticed similar issues with npm and 7zip.

I raised an issue on Oneget:https://github.com/OneGet/oneget/issues/169. The folks over there feel that Chocolatey is returning multiple results in this situation when it should return one - most likely, the package that matched the name exactly.

**Comment by ferventcoder** [comment]
I agree. The problem here is that we don't have a provider available yet for OneGet, so YMMV on whatever it is that you are using, likely the CTP built by MS in April 2014. I would suggest you use the official choco client for now and wait for us to announce official OneGet support (which we expect to have sometime this summer). We'll announce it in the newsletter. Thanks!

**Comment by JohnLudlow** [comment]
Ok, will do

Waiting patiently....
Waiting patiently....
Waiting patiently....

Ready yet? ;P

Thanks!

PS: Do you want to close this issue or leave it open for now?

**Comment by ferventcoder** [comment]
Let's keep it open in case other folks come across this issue. :)

**Comment by JohnLudlow** [comment]
Sounds like a plan

Thanks again
